cli/params.yaml

```diff
@@ -0,0 +1,34 @@
+params:
+  - name: s3-bucket-name
+    value: rhoai-abdalla-pipelines
```

Comment:
This seems quite specific to you. Is this file even expected to be committed here?

Reply:
Indeed it wasn't supposed to be committed. I changed it to be a sample file that's under `cli/examples` and added instructions in the `docs/cli.md` documentation.